[PATCH 35/37] board: mediatek: fix NULL upgrade handler of nor+emmc

Remove unexpected unused macro for bl upgrading

Change-Id: I7c3af378259e90e52afb43cd62501e4046c49e73
---
 board/mediatek/common/bootmenu_snor_emmc.c | 12 ------------
 1 file changed, 12 deletions(-)
--- a/board/mediatek/common/bootmenu_snor_emmc.c	2025-09-08 15:23:17.917688572 +0800
+++ b/board/mediatek/common/bootmenu_snor_emmc.c	2025-09-08 15:23:17.881687020 +0800
@@ -22,11 +22,7 @@
 		.abbr = "fip",
 		.env_name = "bootfile.fip",
 		.validate = generic_validate_fip,
-#if defined(CONFIG_FIP_IN_SPI_NOR)
 		.write = generic_mtd_write_fip,
-#elif defined(CONFIG_FIP_IN_EMMC)
-		.write = generic_mmc_write_fip,
-#endif
 		.post_action = UPGRADE_ACTION_CUSTOM,
 		.do_post_action = generic_invalidate_env,
 	},
@@ -36,11 +32,7 @@
 		.abbr = "bl31",
 		.env_name = "bootfile.bl31",
 		.validate = generic_validate_bl31,
-#if defined(CONFIG_FIP_IN_SPI_NOR)
 		.write = generic_mtd_update_bl31,
-#elif defined(CONFIG_FIP_IN_EMMC)
-		.write = generic_mmc_update_bl31,
-#endif
 		.post_action = UPGRADE_ACTION_CUSTOM,
 	},
 	{
@@ -48,11 +40,7 @@
 		.abbr = "bl33",
 		.env_name = "bootfile.bl33",
 		.validate = generic_validate_bl33,
-#if defined(CONFIG_FIP_IN_SPI_NOR)
 		.write = generic_mtd_update_bl33,
-#elif defined(CONFIG_FIP_IN_EMMC)
-		.write = generic_mmc_update_bl33,
-#endif
 		.post_action = UPGRADE_ACTION_CUSTOM,
 		.do_post_action = generic_invalidate_env,
 	},
